<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaiduMap</class>
 <widget class="QMainWindow" name="BaiduMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QPushButton" name="pBtn_ZoomDown">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>560</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/map/Map/img/zoom_down.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_ZoomUp">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>510</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/map/Map/img/zoom_up.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_Close">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>170</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
